RomeALincei/ANLcr62.xml

```diff
@@ -0,0 +1,445 @@
+                                            ዕለተ፡ አዘዞ፡ ከመ፡ ይትጋብኡ፡ <cb n="b"/> ሠራዊቱ።
+                                        </incipit>
+                                    </msItem>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i3.8">
+                                    <locus from="132va" to="140vb"/>
+                                    <title ref="LIT7331SaySelColl#Sunday" type="complete"/>
+                                    
+                                    <msItem xml:id="ms_i3.8.1">
+                                        <locus from="132va" to="139rb"/>
+                                        <title ref="LIT7331SaySelColl#PrayerEgzieya" type="complete"/>
+                                        <note>The text of the prayer for Friday seems to be different from the one
+                                            transmitted in <ref type="mss" corresp="BAVet97"/>.</note>
+                                        <incipit xml:lang="gez">
+                                            ጸሎት፡ ዘዕለተ፡ እሁድ፡ ይትባረክ፡ እግዚአብሔር፡ እግዚእየ፡ ወአምላኪየ፡ እ<cb n="b"/>ስእለከ፡ በፍቁር፡ 
+                                            ወልድከ፡ 
+                                            እግዚእነ፡ ወአምላክነ፡ ወመድኃኒነ፡ በአስማቲከ፡ ቅዱሳት፡ ከመ፡ ትስማእ፡ ጸሎትየ፡ ወአስተብቍዖትየ፡
+                                        </incipit>
+                                    </msItem>
+                                    
+                                    <msItem xml:id="ms_i3.8.2">
+                                        <locus from="139rb" to="140vb"/>
+                                        <title ref="LIT7318MTMonthly" type="complete"/>
+                                        <incipit xml:lang="gez">
+                                            ተአምራቲሆሙ፡ <gap reason="ellipsis"/> ወአሐተ፡ ዕለተ፡ ቦኡ፡ ውስተ፡ አርያም፡ ወከሠቱ፡ አፉሆሙ፡ 
+                                            አብ፡ ወወልድ፡ ወ<pb n="139v"/><cb n="a"/>መንፈስ፡ ቅዱስ፡ 
+                                            ወይቤሉ፡ ኵሉ፡ ሕዝብ፡ ያጽምዕ፡ በበወርኁ፡ ወይከውን፡ ዕለተ፡ በዓልነ፡ ከመ፡ ዕለተ፡ ትንሣኤ።
+                                        </incipit>
+                                    </msItem>
+                                </msItem>
+                            </msItem>
+        
+                    </msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="144">144</measure>
+                                    <measure unit="quire" quantity="14">14</measure>
+                                    <measure unit="page" type="blank">20</measure>
+                                    <locus from="1v" to="4v"/>
```

Comment:
i do not understand the loci, to what do they refer?

Reply:
these are the blank pages

Reply:
> these are the blank pages

yes, this was my guess, but then they do not result into 20, if my calculation is correct

Reply:
right, 21 should be correct?

Reply:
I would say